### DISCUSSION TITLE
Updating the database version takes more than 3 hours

### DISCUSSION THREAD

**Discussion by q1LLa** [supabase/supabase#20484] [Questions]
Hi. I clicked on the "update database version" button and this process has already been going on for more than 3 hours. 

What should I do?

**Comment by q1LLa** [comment]
![chrome_umv8gUyz4C](https://github.com/supabase/supabase/assets/15686231/77d75a1e-26f8-4425-a16d-dbf126ee1765)
The database is very small and weighs almost nothing.

**Reply by GaryAustin1** [reply]
Hard refresh your browser.  If still there then contacting support is best option.

**Reply by q1LLa** [reply]
The browser is fine, I tried it on other devices - it's the same there.
